Now there is a new error in spdlog compilation:
~~~
 D:\a\drake-lite-cmake\drake-lite-cmake\spdlog\include\spdlog/details/fmt_helper.h(37,43): error C3083: 'internal': the symbol to the left of a '::' must be a type [D:\a\drake-lite-cmake\drake-lite-cmake\spdlog\build\spdlog.vcxproj]
D:\a\drake-lite-cmake\drake-lite-cmake\spdlog\include\spdlog/details/fmt_helper.h(37,53): error C2039: 'count_digits': is not a member of 'fmt' [D:\a\drake-lite-cmake\drake-lite-cmake\spdlog\build\spdlog.vcxproj]
~~~
Probably it is the effect of some update of MSVC or similar.

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -123,7 +123,7 @@ jobs:
         # spdlog
         git clone https://github.com/gabime/spdlog
         cd spdlog
-        git checkout v1.6.0
+        git checkout v1.7.0
         mkdir -p build
         cd build
         cmake -A x64 -DSPDLOG_BUILD_SHARED:BOOL=ON -DSPDLOG_BUILD_EXAMPLE:BOOL=OFF -DSPDLOG_FMT_EXTERNAL:BOOL=ON -DBUILD_TESTING:BOOL=OFF -DCMAKE_TOOLCHAIN_FILE=${VCPKG_INSTALLATION_ROOT}/scripts/buildsystems/vcpkg.cmake -DCMAKE_INSTALL_PREFIX=${GITHUB_WORKSPACE}/install ..

